src/reducers/profilesReducer.ts

```diff
@@ -48,7 +48,8 @@ export function newProfile(ethereumAccountAddress: string): IProfileState {
 
 const initialState: IProfilesState = { threeBox: null, threeBoxSpace: null };
 
-const profilesReducer = (state = initialState, action: any) => {
+//eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.

Reply:
As we spoke, this one I leave for you as you're working on this file and you've a solution for this.